src/hooks/useIsLoading.js

```diff
@@ -0,0 +1,11 @@
+import React, { useEffect, useState } from 'react';
+
+export const useIsLoading = (loadingsArray = []) => {
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    setIsLoading(loadingsArray.some((item) => item));
+  }, [...loadingsArray]);
+
+  return isLoading;
```

Comment:
Return the object with isLoading, { isLoading }, then it will be easy to add something if we will have such scenario.